plugin/gitgutter.vim

```diff
@@ -118,7 +118,8 @@ command! -bar GitGutterBufferDisable call gitgutter#buffer_disable()
 command! -bar GitGutterBufferEnable  call gitgutter#buffer_enable()
 command! -bar GitGutterBufferToggle  call gitgutter#buffer_toggle()
 
-command! -bar GitGutterQuickFix call gitgutter#quickfix()
+command! -bar GitGutterQuickFix call gitgutter#quickfix(v:false)
+command! -bar GitGutterQuickFixCurrentFile call gitgutter#quickfix(v:true)
```

Comment:
Please could you use `0`/`1` instead of `v:false`/`v:true` for more backward compatibility?  They were introduced in 7.4.1154 which is quite old but I try to support 7.4 altogether.

Reply:
done.